src/cryptography/hazmat/backends/openssl/rsa.py

```diff
@@ -559,6 +565,62 @@ def private_numbers(self):
             )
         )
 
+    def as_bytes(self, encoding, fmt, encryption_algorithm):
```

Comment:
Unless `flake8` complains, I think it would be preferable.

On Fri, Feb 27, 2015 at 1:03 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/backends/openssl/rsa.py
> <https://github.com/pyca/cryptography/pull/1503#discussion_r25538744>:
>
> > @@ -559,6 +565,62 @@ def private_numbers(self):
> >              )
> >          )
> >
> > +    def as_bytes(self, encoding, fmt, encryption_algorithm):
>
> Yep. If we're comfortable with it shadowing the builtin for this method I
> am okay with using format though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1503/files#r25538744>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084